ArduCopter/mode_auto.cpp

```diff
@@ -1915,9 +1912,9 @@ bool ModeAuto::verify_payload_place()
             return false;
         }
         // we have a valid calibration.  Hopefully.
+        // todo: Add filter as single sample is vulnerable to noise
         nav_payload_place.hover_throttle_level = current_throttle_level;
-        const float hover_throttle_delta = fabsf(nav_payload_place.hover_throttle_level - motors->get_throttle_hover());
-        gcs().send_text(MAV_SEVERITY_INFO, "hover throttle delta: %f", static_cast<double>(hover_throttle_delta));
+        gcs().send_text(MAV_SEVERITY_INFO, "payload hover throttle: %f", static_cast<double>(nav_payload_place.hover_throttle_level));
```

Comment:
This change in reporting is probably a good one but if we're going to change the message let's also prefix "PayloadPlace: " to it to be consistent with other reporting we do.